services/suggest.integration.js

```diff
@@ -0,0 +1,145 @@
+'use strict'
```

Comment:
I'm not sure where we landed with #2845, and I know we use this file naming pattern for other integration tests (under `core` for example) but if we end up renaming `services` (files, classes, and/or otherwise) to something `integration` related as part of #2845, I could see things under `services` getting confusing. 

This is probably good as is, and we can rename/refactor as needed as part of the #2845 work, but just wanted to jot this down